crimsobot/cogs/cringo.py

```diff
@@ -170,6 +170,7 @@ def generate_mismatch_embed(self) -> discord.Embed:
         return embed
 
     def remove_from_game(self, user: discord.User) -> discord.Embed:
+        CringoGame.all_players.discard(user)
```

Comment:
side rant: i wish python had a better way of doing this so the class doesn't need to be aware of it's own name... and no I'm not advocating for the class dunder, that's even more gross lol.